src/serena/util/file_system.py

```
@@ -52,7 +52,11 @@ def scan_directory(
                     entry_path = entry.path
 
                     if rel_base:
-                        result_path = os.path.relpath(entry_path, rel_base)
+                        try:
+                            result_path = os.path.relpath(entry_path, rel_base)
+                        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)